indexMap.opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "2.0"
+maintainer: "hez@0ok.org"
+authors: [ "Hezekiah M. Carty <hez@0ok.org>" ]
+license: "MIT"
+homepage: "https://github.com/hcarty/indexmap"
+bug-reports: "https://github.com/hcarty/indexmap/issues"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.0.0"}
+  "bigarray"
```

Comment:
```suggestion
```
I don't think this is needed.  If it is, maybe it should be `base-bigarray` instead?